### PR TITLE
Adding Voipbuster SMS support

### PR DESCRIPTION
**Pull request by JdeJongATS**
Reused code from other provider

### CODE REVIEW

File: lib/Command/Configure.php [hunk 1]
```diff
@@ -217,6 +217,26 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setDid($did);
 				break;
 
+			case 'voipbuster':
+				$config->setProvider($provider);
+		
+				/** @var VoipMsConfig $providerConfig */
```
Comment (ChristophWurst) [inline lib/Command/Configure.php:223]: import of this class is missing

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Looks good otherwise

File: lib/Command/Configure.php (hunk 1 above)
Comment (JdeJongATS) [reply lib/Command/Configure.php:223]: > import of this class is missing

My very first pull request; and a complete noob in GitHub ;) 
What do you mean? I forgot to sync the ProviderFactory.php aswell from my nextcloud instance, which handles the setProvider statement.

Comment (ChristophWurst) [reply lib/Command/Configure.php:223]: No problem :)

There is no `use ...` statement at the head of the file for this class. See how the other providers import their config.

Please push the provider factory change as well

Comment (JdeJongATS) [reply lib/Command/Configure.php]: Added the class definition in the head of the file.

**Review by ChristophWurst — APPROVED** [review]
Looks good :)

One last thing: could you try to squash your five commits into one? There are some guides like https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git oneline, hope that helps if you're doing this for the very first time.

Otherwise I can help and do it for you, no worries

**Comment by JdeJongATS** [comment]
@ChristophWurst hopefully this works. It seems it's merged in one new commit (189360c).

**Pull request closed without merging**